src/broker/Data.cc

```diff
@@ -955,6 +983,31 @@ broker::expected<broker::data> val_to_data(const Val* v)
 			rval.emplace_back(move(*item));
 			}
 
+		return {std::move(rval)};
+		}
+	case TYPE_LIST:
+		{
+		// We don't really support lists on the broker side.
+		// So we just pretend that it is a vector instead.
+		auto list = v->AsListVal();
+		broker::vector rval;
+		rval.reserve(list->Length());
+
+		for ( auto i = 0u; i < list->Length(); ++i )
```

Comment:
```suggestion
		for ( auto i = 0; i < list->Length(); ++i )
```
signed/unsigned comparison compiler warning